app/build.gradle

```diff
@@ -1,83 +1,85 @@
 apply plugin: 'com.android.application'
-
-configurations.all {
-    setTransitive(true)
-}
-
-// For the RightMesh library.
-repositories {
-    google()
-    maven {
-        url "https://research.rightmesh.io/artifactory/libs-release-local"
-        credentials {
-            username artifactory_app_username
-            password artifactory_app_password
-        }
-    }
-}
-
+// NOTE: you MUST disable configureOnDemand for your compiler.
+// To do this in Android Studio: https://stackoverflow.com/a/49994951/4082582
 buildscript {
     ext {
-        artifactory_app_username = "insertyourusername"
-        artifactory_app_password = "insertyourpasswordhash"
-        artifactory_app_key = "insertyourkey"
+        // define these variables in your personal ~/.gradle/gradle.properties
+        // for example:
+        /*
+            rightmesh_artifactory_username=<username>
+            rightmesh_artifactory_password=<hash of your developer portal password>
+            rightmesh_reflect_key=<some key that you have access to>
+        */
+        artifactory_app_username = "${rightmesh_artifactory_username}"
+        artifactory_app_password = "${rightmesh_artifactory_password}"
+        artifactory_app_key = "${rightmesh_reflect_key}"
+
+        maven_repo = "https://research.rightmesh.io/artifactory/libs-local"
     }
     repositories {
-        google()
-        // For the RightMesh Gradle Plugin
         maven {
-            url "https://research.rightmesh.io/artifactory/libs-release-local"
+            url maven_repo
             credentials {
                 username artifactory_app_username
                 password artifactory_app_password
             }
         }
-
-        // For RetroLambda.
+        google()
         mavenCentral()
+        jcenter()
     }
     dependencies {
         classpath 'io.left.rightmesh:rightmesh-plugin:1.6'
-        classpath 'me.tatarka:gradle-retrolambda:3.7.0'
     }
 }
 apply plugin: 'io.left.rightmesh.rightmesh-plugin'
 preBuild.dependsOn("rightmesh")
 
+
+repositories {
+    mavenCentral()
+    jcenter()
+    google()
+    maven {
+        url maven_repo
+        credentials {
+            username artifactory_app_username
+            password artifactory_app_password
+        }
+    }
+}
+
 android {
     compileSdkVersion 27
-    buildToolsVersion "27.0.3"
+    buildToolsVersion '27.0.3'
     defaultConfig {
-        applicationId "io.left.reflect"
+        applicationId "io.left.hellomesh"
```

Comment:
This is the `reflect` repo, the applicationId should still be `io.left.reflect`

Reply:
Moved back to reflect